File: app/src/main/java/Start/PagerAdapter.java
package com.example.ty395.google_map.Start;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

public class PagerAdapter extends FragmentStatePagerAdapter {
    public PagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int i) {
        switch (i) {
            case 0:
                return new MapFragment();
            case 1:
                return new RouteFragment();
            case 2:
                return new PlaceFragment();
            case 3:
                return new StartFragment();
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return 4;
    }
}

File: app/src/main/java/Start/PagerAdapter.kt
package com.example.ty395.google_map.Start

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

class PagerAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

    override fun getItem(i: Int): Fragment? {
        when (i) {
            0 -> return MapFragment()
            1 -> return RouteFragment()
            2 -> return PlaceFragment()
            3 -> return StartFragment()
            else -> return null
        }
    }

    override fun getCount(): Int {
        return 4
    }
}
